libs/scripts/accessibility.js

```diff
@@ -97,6 +97,18 @@ export const setDialogAndElementAttributes = ({ element, title }) => {
   element.closest('.dialog-modal')?.setAttribute('aria-label', title);
 };
 
+export function setListRoles() {
+  const selector = ':is(.section[class*="-up"]:has(.action-item, .icon-block), .scroller):not(:has(h1, h2, h3, h4, h5, h6))';
```

Comment:
Looking at the selector, I'm thinking it might need a bit more protection - are the action-item/icon-block elements usually direct children of the section? Judging by the the `[...parent.children]` on L104, I'd assume so. If so, maybe a `>` child combinator could make things more specific, to avoid applying this rule to sections that happen to have a single action item _somewhere_. Speaking of 'a single action item', should we check that there are multiple in order to apply the rule? Something as simple as `:nth-child(2)` within the `:has` braces could account for this as well.